node_test/rpc_test.go

```diff
@@ -99,6 +98,7 @@ func executeNRpcTest(t *testing.T, connectionType transport.TransportType, n int
 	for i := 0; i < n; i++ {
 		sk := `000000000000000000000000000000000000000000000000000000000000000` + strconv.Itoa(i+1)
 		actors[i] = ta.Actor{
+			// PrivateKey: common.Hex2Bytes(sk),
```

Comment:
✂️

Reply:
Fixed in https://github.com/statechannels/go-nitro/pull/1747/commits/f9a9b81b920d412edb92208ad0735c8b674bd0d9